src/system/SystemPacketBuffer.h

```diff
@@ -474,6 +475,14 @@ class DLL_EXPORT PacketBufferHandle
      */
     bool IsNull() const { return mBuffer == nullptr; }
 
+    /**
+     * Test whether the PacketBuffer owned by this PacketBufferHandle has unique ownership.
+     *
+     * @return \c true if the PacketBuffer owned by this PacketBufferHandle is solely owned; return \c false if
```

Comment:
comment seems wrong: this returns true if refcount > 1 (i.e. NOT solely owned) and false otherwise.

How about a name of 'HasSoleOwnership' to make it obvious what the method does without needing to read coments.

That being said, I believe we should take the approoach of cloning instead of adding lwip-specific code that relies on packetbuffer internal reference counts.

Reply:
Acked